build.sh

```diff
@@ -119,11 +119,7 @@ then
   exit 0
 fi
 
-# First build the analyzers, for use in everything else.
 log_build_action "(Start) build.sh"
-log_build_action "Building analyzers"
-
-dotnet publish -nologo -clp:NoSummary -v quiet -c Release -f netstandard2.0 tools/Google.Cloud.Tools.Analyzers
```

Comment:
I think we were not using them at all, we are building for `netstandard2.0` here but from the next commit it seems that we were looking for them on the `netstandard1.3` path. Unless I'm missing something.